x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -197,6 +203,30 @@ export class AlertsClient<
     return { hits, total };
   }
 
+  public async untrackAlertIdByIndices(alertId: string, indices: string[]) {
+    const esClient = await this.options.elasticsearchClientPromise;
+    for (const index of indices) {
+      try {
+        await esClient.updateByQuery({
```

Comment:
Should we check the the reponse body for `updated` and see if match the number of alert that we want to update